zap/src/main/java/org/zaproxy/zap/extension/ascan/ActiveScan.java

```diff
@@ -256,6 +272,10 @@ public void scannerComplete(int id) {
         return null;
     }
 
+    public FilterMessageTableModel getFilterMessageTableModel() {
```

Comment:
Might worth leaving as package accessible for now.

Reply:
ok